bemani/backend/iidx/pendual.py

```diff
@@ -1185,6 +1190,35 @@ def handle_IIDX22pc_request(self, request: Node) -> Optional[Node]:
             elif timeshift_override == 3:
                 common_timeshift_phase.set_attribute('phase', '2')
 
+            root.add_child(Node.void('eaappli_expert'))
+
+            cc_collabo_event = Node.void('cc_collabo_event')
+            root.add_child(cc_collabo_event)
+            if ccxbm_on:
+                cc_collabo_event.set_attribute('phase', "1")
+                root.add_child(Node.void('cc_collabo_license'))
+            else:
+                cc_collabo_event.set_attribute('phase', "0")
+
+            tohoisfumo = Node.void('toho_remix')
+            root.add_child(tohoisfumo)
+            tohoisfumo.set_attribute('phase', "2")
+
+            cm = Node.void("cm")
+            root.add_child(cm)
+            cm.set_attribute('compo', 'cm_event3')
+            cm.set_attribute('folder', 'cm_event3')
+            cm.set_attribute('id', "0")
+
+            #Ranking details in title screen
+            rankingcharts = []
+            for (mid, plays) in self.data.local.music.get_hit_chart(self.game, self.music_version, 20):
+                rankingcharts.append(mid)
+            root.add_child(Node.u16_array('monthly_mranking', rankingcharts))
+            root.add_child(Node.u16_array('total_mranking', rankingcharts))
+            ##need to add in the top 20 rankings for month and total network
```

Comment:
Is that not what the above section does? Except for it doesn't really do it for the last month.

Reply:
need to fix this with the proper values